### PR TITLE
(#869) Update to make doc consistent

### LINKED ISSUE #869: Deployment Plan Examples should be Extended
## What New Or Updated Would You Like To See?

There is currently a stub of an article for deployment examples. It would be great, now we have some examples, to complete the loop.

## Why Is It Needed?

It fills in a gap!

## Additional Context?
N/A

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Updated a recent doc change to make it consistent with other docs.

Note that I didn't update the button to have `View ``.json``` as I didn't know if it would break the button.

## Motivation and Context
Just consistency.

## Testing

* [ ] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [x] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #869 (this is closed but relates to the changes made there)

### CODE REVIEW

File: input/shared/import-deployment-plan.txt
Comment (pauby) [inline input/shared/import-deployment-plan.txt:1]: I removed the link from here as they don't work on deployment-plans page they are displayed on.

**Comment by pauby** [comment]
@st3phhays @gep13 Can you confirm that the `RedirectFrom` is correct? I copied it from another page.

File: input/en-us/central-management/usage/examples/deployment-plans.md [hunk 1]
```diff
@@ -325,9 +325,9 @@ Save the following as a json file to import as a Chocolatey Central Management D
 ```
 </div>
 
-##### Upgrade the Chocolatey for Business client components to the latest version.
+##### Upgrade the Chocolatey for Business Client Packages to the Latest Version.
 
-Save the following as a json file to import as a Chocolatey Central Management Deployment Plan:
+Save the following as a `.json` file to import as a Chocolatey Central Management Deployment Plan:
 
 <button class="btn btn-success btn-collapse collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#absoluteLatestJson" aria-expanded="false" aria-controls="absoluteLatestJson">View json</button>
```
Comment (st3phhays) [inline input/en-us/central-management/usage/examples/deployment-plans.md:332]: For all these buttons, shouldn't it be `View JSON` ?

If you're not referring to it as a file extension like `.json`, then the correct way would be to capitalize it.

**Comment by st3phhays** [comment]
> @st3phhays @gep13 Can you confirm that the `RedirectFrom` is correct? I copied it from another page.

I have ran this locally and can confirm that the redirect works as expected.

File: input/en-us/central-management/usage/examples/deployment-plans.md (hunk 1 above)
Comment (gep13) [reply input/en-us/central-management/usage/examples/deployment-plans.md:332]: Yes, I agree, I will get this fixed up.

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**